src/toM3u8.js

```diff
@@ -89,6 +91,11 @@ export const formatAudioPlaylist = ({ attributes, segments, sidx }) => {
     playlist.sidx = sidx;
   }
 
+  if (isAudioOnly) {
```

Comment:
if we are an audio only playlist, we still want subtitles and audio groups to work correctly.